project.clj

```diff
@@ -15,11 +14,11 @@
                  [duct/server.http.jetty "0.2.1"]
```

Comment:
are we using the duct module or the regular jetty ring adapter?

Reply:
We are using duct, it has the :handler key extension for jetty